app/src/main/java/org/mozilla/fenix/IntentReceiverActivity.kt

```diff
@@ -79,6 +84,24 @@ class IntentReceiverActivity : Activity() {
                 // from a session that the user already "erased".
                 intent.flags and Intent.FLAG_ACTIVITY_LAUNCHED_FROM_HISTORY == 0
             }
+            intent.action == ACTION_OPEN_TAB || intent.action == ACTION_OPEN_PRIVATE_TAB -> {
+                intent.setClassName(applicationContext, HomeActivity::class.java.name)
+                val startPrivateMode = (intent.action == ACTION_OPEN_PRIVATE_TAB)
+                if (startPrivateMode) {
+                    intent.putExtra(
+                        HomeActivity.OPEN_TO_SEARCH,
+                        StartSearchIntentProcessor.PRIVATE_BROWSING_STATIC_SHORTCUT_PRIVATE
```

Comment:
Can you help me understand the difference between `                        StartSearchIntentProcessor.PRIVATE_BROWSING_STATIC_SHORTCUT_PRIVATE` and `                        StartSearchIntentProcessor.PRIVATE_BROWSING_STATIC_SHORTCUT`? Their names seem a bit confusing to me.

Reply:
I'll change the names to make it clearer.